Pod/Classes/Internal/SEGSegmentIntegration.m

```diff
@@ -528,6 +532,17 @@ - (void)applicationWillTerminate
     }];
 }
 
+-(void)applicationDidFinishLaunching
+{
+    [self beginBackgroundTask];
+    // We will check if there is a previous build of the app where traits/queue was stored in NSQ
+    // If there is then we will store the existing queue and traits in the new location
+    if ([self hasPreviousVersion]) {
```

Comment:
Why do we need to check for version explicitly? Can we just check the existence of `SEGQueue` and `SEGTraits` key directly in `NSUserDefaults` and if they exist then migrate them?